doc/refman/RefMan-pre.tex

```diff
@@ -1170,31 +1171,34 @@ \section*{Credits: version 8.6 (Stronger, Better, Faster Rooster)}
 Claret, Guillaume Melquiond, Matthieu Sozeau, Enrico Tassi and others. A
 list of packages is now available at \url{https://coq.inria.fr/opam/www/}.
 
-Packaging tools and software development kits were prepared by Maxime
-Dénès, Michael Soegtrop and Enrico Tassi for Windows, and Maxime Dénès
-and Matthieu Sozeau for MacOS X. Packages are now regularly built on the
-continuous integration server.
+Packaging tools and software development kits were prepared by Michael
+Soegtrop with the help of Maxime Dénès and Enrico Tassi for Windows, and
+Maxime Dénès and Matthieu Sozeau for MacOS X. Packages are now regularly
+built on the continuous integration server.
 
 Matej Košík maintained and greatly improved the continuous integration
 setup and the testing of {\Coq} contributions. He also contributed many
 API improvement and code cleanups throughout the system.
 
-General maintenance during part or whole of this period has been done by
-Pierre Boutillier, Pierre Courtieu, Maxime Dénès, Hugo Herbelin, Pierre
-Letouzey, Guillaume Melquiond, Pierre-Marie Pédrot, Matthieu Sozeau,
-Arnaud Spiwack, Enrico Tassi, Bruno Barras, Yves Bertot,
-Frédéric Besson, Assia Mahboubi and Yann Régis-Gianas. The development
-process was coordinated by Matthieu Sozeau with the help of Maxime
-Dénès, who was also in charge of the release process.
+The contributors for this version are C.J. Bell, Yves Bertot, Frédéric
+Besson, Tej Chajed, Pierre Courtieu, Maxime Dénès, Ricky Elrod, Jason
+Gross, Hugo Herbelin, Emilio Jesus Gallego Arias, Jacques-Henri Jourdan,
+Matej Košík, Xavier Leroy, Pierre Letouzey, Gregory Malecha, Cyprien
+Mangin, Erik Martin-Dorel, Guillaume Melquiond, Pierre-Marie Pédrot,
+Lionel Rieg, Gabriel Scherer, Matthieu Sozeau, Arnaud Spiwack, Paul
+Steckler, Laurent Théry, Enrico Tassi, Nickolai Zeldovich, Théo
+Zimmermann and Daniel de Rauglaudre. The development process was
+coordinated by Hugo Herbelin and Matthieu Sozeau with the help of
+Maxime Dénès, who was also in charge of the release process.
```

Comment:
Indeed. I had Sébastien and Pierre B in my hand-crafted list. Fixed Daniel's place. For the record, I'm using: git log --since "JAN 1 2016" --until "DEC 5 2016" --pretty=format:"%an," | sort -k 2 | uni